### ISSUE TITLE
[Docs] some questions on how the package works

### ISSUE THREAD

**Issue by ctf0** [nWidart/laravel-modules#1161]
first thanks a lot for this package, been having the same idea for quite some time but never figured out how exactly to tackle it, so big thanks for creating this ❤️ .

now for the original matter.

- what `modules_statuses.json` does ? & how the module activator works ?
- why do have `module.json` & `composer.json` under each module especially that both have a very similar structure ?
- why each module config have a default key `name` ? what is it for ? does removing it cuz issues to the module registration or not ?
- what `cache` under `config/modules.php` is for ?

**Comment by FoksVHox** [comment]
**What does `modules_statusses.json`**
It dictates which modules that are active, and which aren't 

**Why do have `module.json` &` composer.json` under each module especially that both have a very similar structure**
The `composer.json` defines the composer packages used in the module. 
The `module.json` defines general things about the module.

**Why each module config have a default key `name`**
It is the module's name.

**What `cache `under `config/modules.php` is for**
Basically caches stuff, just like Laravel

**Comment by ctf0** [comment]
@FoksVHox i was looking more for an in depth details but still thanks.
